Common/Orders/Fees/AlphaStreamsFeeModel.cs

```diff
@@ -0,0 +1,163 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using QLNet;
+using QuantConnect.Data.Fundamental;
+using QuantConnect.Orders.Fills;
+using QuantConnect.Securities;
+
+namespace QuantConnect.Orders.Fees
+{
+    public class AlphaStreamsFeeModel : FeeModel
+    {
+
+        private readonly decimal _forexCommissionRate = 0.000002m;
+        private decimal _liborRate;
+
+        private readonly Dictionary<string, EquityFee> _equityFee =
+            new Dictionary<string, EquityFee>();
+
+        private readonly Dictionary<string, CashAmount> _futureFee =
+            //                                                               Commission plus clearing fee
+            new Dictionary<string, CashAmount> { { Market.USA, new CashAmount(0.4m + 0.1m, "USD") } };
+
+        private readonly Dictionary<string, CashAmount> _optionFee =
+            //                                                               Commission plus clearing fee
+            new Dictionary<string, CashAmount> { { Market.USA, new CashAmount(0.4m + 0.1m, "USD") } };
+
+        public AlphaStreamsFeeModel(decimal liborRate = 0.024m)
+        {
+            _liborRate = Math.Abs(liborRate);
+            _equityFee.Add(Market.USA, new EquityFee("USD", 0.004m + _liborRate, 0));
+        }
+        /// <summary>
+        /// Gets the order fee associated with the specified order. This returns the cost
+        /// of the transaction in the account currency
+        /// </summary>
+        /// <param name="parameters">A <see cref="OrderFeeParameters"/> object
+        /// containing the security and order</param>
+        /// <returns>The cost of the order in units of the account currency</returns>
+        public override OrderFee GetOrderFee(OrderFeeParameters parameters)
+        {
+            var order = parameters.Order;
+            var security = parameters.Security;
+
+            // Option exercise for equity options is free of charge
+            if (order.Type == OrderType.OptionExercise)
+            {
+                var optionOrder = (OptionExerciseOrder)order;
+
+                if (optionOrder.Symbol.ID.SecurityType == SecurityType.Option &&
+                    optionOrder.Symbol.ID.Underlying.SecurityType == SecurityType.Equity)
+                {
+                    return OrderFee.Zero;
+                }
+            }
+
+            decimal feeResult;
+            string feeCurrency;
+            var market = security.Symbol.ID.Market;
+            switch (security.Type)
+            {
+                case SecurityType.Forex:
+                    // get the total order value in the account currency
+                    var totalOrderValue = order.GetValue(security);
+                    var fee = Math.Abs(_forexCommissionRate * totalOrderValue);
```

Comment:
nit - if `order.GetValue` has the potential to be negative, then I think it makes more sense to place the `Math.Abs` around that instead of around this multiplication operation.